lbz/dev/server.py

```diff
@@ -56,9 +60,9 @@ def _get_route_params(self, org_path: str) -> Tuple[Union[str, None], Union[dict
                         acc += 1
                 if len(path) == acc:
                     return org_route, params
-        return None, None
+        raise ValueError
```

Comment:
Please place here an error message to say directly what happened - it will simplify resolving problems in the future 😉